nservicebus/nhibernate/accessing-data_direct-access_nhibernate_[6,7).partial.md

```diff
@@ -0,0 +1,7 @@
+Note that in Version 5 and below, `ISession` could be injected directly into the handlers. This behavior has changed in NServiceBus Version 6 as internal components of NServiceBus are no longer accessible from the [container](/nservicebus/containers/). This approach can still be used on Version 5 and below.
```

Comment:
@SzymonPobiega I split this further apart into [4,6), [6,7), and [7,] version ranges.